CMakeLists.txt

```diff
@@ -170,7 +170,7 @@ endif()
 target_compile_definitions(FLUID_DECOMPOSITION INTERFACE EIGEN_MPL2_ONLY=1)
 
 if(APPLE)
-  target_compile_definitions(FLUID_DECOMPOSITION INTERFACE EIGEN_USE_BLAS=1)
+  # target_compile_definitions(FLUID_DECOMPOSITION INTERFACE EIGEN_USE_BLAS=0)
```

Comment:
I think just delete this line. Individuals can always restore using BLAS from the command line (i.e. by passing `-DEIGEN_USE_BLAS=1`).